content/departments/product-engineering/engineering/career-development/talent-review-process.md

```diff
@@ -64,13 +64,19 @@ Our [career development framework](framework.md) defines the levels that exist a
 
 ## Promotion requests
 
-Promotion requests are made immediately following the [talent review calibration meeting](#calibration-meeting).
+During the [talent review calibration meeting]](#calibration-meeting) managers indicate which teammates they would like to considered for promotion. These teammates are then evaluated against their peers at the level they might get promoted to so that the manager can confirm they are indeed performing at the right level for promotion.
 
-The Engineering Manager will complete a promotion request using the [promotion request template](https://docs.google.com/document/d/1DHyPtIooi0J2426iemzzTqLle9UBasrqLZPGET1O7Sc/edit) and submit it for approval.
+If the manager believes the candidate is ready for promotion, a promotion request is made following the [talent review calibration meeting](#calibration-meeting).
```

Comment:
On the call, we chatted around discretion on timing to submit a promotion request – managers might want to get feedback from their peers to calibrate on how close an individual is to promo but they might want to see the completion of a project or milestone before following through. Makes sense to add a small blurb on that?

Reply:
@yinquanteo I'm wondering if that is too much granularity in the process description.

I think managers should not put candidates forward for promotion if they don't have the signals yet that they are ready (i.e. wanting to wait for a project to be completed).

Reply:
Yea, perhaps. I guess the thing I was hoping to address is the concern I heard from EMs around having to wait another 6 months (if we move to bi-annual cadence) before putting someone up for promo when someone is close enough to the next level. Said in another way, if we moved to a bi-annual cadence, the current guidance does incentivize EMs to promote folks earlier than when they are full ready just to meet the calibration cut off because these folks are close enough to the next level.

Reply:
I hear this as an argument to not move to a bi-annual cadence. (Or if we do, _then_ we should make sure not to tie promotions to the calibration cadence.)

Reply:
I agree that if we moved to a bi-annual cadence this would feel too long. 
We might consider an in-between, not formal company calibration, session so we can still keep the quarterly cadence.
If not I think we should reconsider our process including tying promotions to calibration cadence as Chris mentioned.
Until a decision is made on that, and to keep this change a small iteration, I propose we accept the current wording and revisit this when needed.